quiz.rb

```diff
@@ -2,19 +2,60 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+class Dog
+
+  attr_reader :name
+  attr_reader :breed
+  attr_reader :age
+
+  def initialize(name,breed,age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+
+end
+
 # Question 2
 # Create a new Dog object.
 
+parker = Dog.new("Parker","Eskimo Lab","4")
+
 # Question 3
 # Create a Person class, with attributes name, age, and location.
 # It should set all three attributes when you create a new Person.
 # Name and location should be writeable, but age should be read-only.
 
+class Person
+
+  attr_writer :name
+  attr_reader :age
+  attr_writer :location
+
+  def initialize(name,age,location)
+    @name = name
+    @age = age
+    @location = location
+  end
+
+end
+
 # Question 4
 # Create a new Person object using parameters "Dave", 32, and "Ohio".
 # Then, set that Person's location to "Somerville"
 
+dave = Person.new("Dave","32","Ohio")
+dave.set_location("Somerville")
```

Comment:
Almost! Using `attr_accessor` allows us to manipulate an object's properties just like we could with JavaScript, so we don't need to rely on extra methods; just write `dave.location = "Somerville"`.